presto-parser/src/main/java/com/facebook/presto/sql/tree/ExpressionTreeRewriter.java

```diff
@@ -622,12 +622,10 @@ public Expression visitLikePredicate(LikePredicate node, Context<C> context)
 
             Expression value = rewrite(node.getValue(), context.get());
             Expression pattern = rewrite(node.getPattern(), context.get());
-            Expression escape = null;
-            if (node.getEscape() != null) {
-                escape = rewrite(node.getEscape(), context.get());
-            }
+            Optional<Expression> escape = node.getEscape()
+                    .map(e -> rewrite(e, context.get()));
```

Comment:
`e` is not an ideal name. Maybe rename to `escape` and rename outer `escape` to `rewrittenEscape`?